rbac/common/manager/rbac_manager.py

```diff
@@ -13,11 +13,90 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 
+import logging
+import inspect
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from rbac.common.protobuf.rbac_payload_pb2 import RBACPayload
 from rbac.common.user.user_manager import UserManager
 from rbac.common.role.role_manager import RoleManager
+from rbac.common.task.task_manager import TaskManager
+
+LOGGER = logging.getLogger(__name__)
+
+
+def walk_filter(value):
```

Comment:
What is this? Why is it called walk_filter?

Would this work for naming?: 

is_class_in_package(value)
    return str(type(value)).startswith(PREFIX_CLASS_IN_COMMON_PKG)

Reply:
changed to _only_rbac_common_